test/integration/containerized/test_container_management.py

```diff
@@ -70,3 +73,65 @@ def test_cancel_will_remove_container(test_data_dir, container_runtime_installed
     assert not is_running(
         cli, container_runtime_installed, 'ansible_runner_foo_bar'
     ), 'Found a running container, they should have all been stopped'
+
+
+def test_invalid_registry_host(tmp_path):
+    pdd_path = tmp_path / "private_data_dir"
+    pdd_path.mkdir()
+    private_data_dir = str(pdd_path)
+
+    image_name = 'quay.io/kdelee/does-not-exist'
+
+    res = run(
+        private_data_dir=private_data_dir,
+        playbook='ping.yml',
+        settings={
+            'process_isolation_executable': 'podman',
```

Comment:
We're currently using docker for the `integration` test suite, so this is unlikely to pass.